README.md

```diff
@@ -121,6 +121,52 @@ Records can be destroyed using the `destroy` method on a table:
 @table.destroy(record)
 ```
 
+## Command Line Tool
+
+This gem is include a very simple command line tool which can show basic functionality of service.
```

Comment:
"This gem includes a simple command line tool which shows the basic functionality of the service."

Reply:
Done